### PR TITLE
Split German and Dutch “Cigo” newsagents

### PR DESCRIPTION
**Pull request by brawer**
They are separate companies with different owners and different
brand identities. Not sure if they’ve been related at some point
in the past (possibly before 2010); as of 2022, they’re separate.

### CODE REVIEW

**Comment by brawer** [comment]
How to fix the build failure? The index build fails because of duplicate `displayName`, but those duplicates are actually in a different `locationSet`.

File: data/brands/shop/newsagent.json [hunk 1]
```diff
@@ -22,10 +22,21 @@
...
     },
+    {
+      "displayName": "Cigo",
+      "id": "cigo-dbf769",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "brand": "Cigo",
+        "brand:wikidata": "Q113290782",
+        "name": "Cigo",
+        "shop": "newsagent"
```
Comment (LaoshuBaby) [inline data/brands/shop/newsagent.json:33]: Looks like this brand sells both newspaper and tobacco?

图片

**Comment by LaoshuBaby** [comment]
> but those duplicates are actually in a different `locationSet`.

You can change the `displayName` to things like "Cigo (Germany)" or "Cigo (Netherland)"

File: data/brands/shop/newsagent.json (hunk 1 above)
Comment (brawer) [reply data/brands/shop/newsagent.json]: Thanks for noticing! Added tag `tobacco=yes`.

**Comment by brawer** [comment]
Thanks for the review! Made `displayName` unique, and added `tobacco:yes` to the German chain.

**Pull request merged**